src/Horudom.Web.Api/Controller/ActorController.cs

```diff
@@ -33,18 +32,70 @@ public async Task<ActionResult<List<ActorDto>>> GetActors()
 			return Ok(result);
 		}
 
-		[HttpGet("{actor}")]
-		public async Task<ActionResult<List<MovieDto>>> GetMoviesByActor(string actor)
+		[HttpGet("{id}")]
+		public ActionResult<ActorDto> GetActor(long id)
+		{
+			var actor = context.Actors.Where(x => x.Id == id).SingleOrDefault();
+
+			return Ok(actor.ToDto());
+		}
+
+		[HttpGet("{id}/movies")]
+		public async Task<ActionResult<List<MovieDto>>> GetMoviesByActor(long id)
 		{
-			var normalizedActor = actor.NormalizeSearch();
+			var actor = await context.Actors.Where(x => x.Id == id).SingleOrDefaultAsync();
+
+			if (actor == null)
+			{
+				return NotFound($"No {nameof(Actor)} with Id {id} found in database");
+			}
+
 			var moviesByActor = await context.MovieActors
-				.Include(x => x.Movie)
-				.Include(x => x.Actor)
-				.Where(x => x.Actor.NormalizedFirstname.Equals(normalizedActor) || x.Actor.NormalizedLastname.Equals(normalizedActor))
+				.Where(x => x.Actor.Id == id)
```

Comment:
You fetched the actor, use it...
```suggestion
				.Where(x => x.Actor.Id == actor.Id)
```